studio/README.md

```diff
@@ -10,12 +10,15 @@ The easiest way to use Lona Studio is by downloading the prebuilt Mac App binary
 
 ## Building from Source
 
-First, make sure you have [`bundler`](http://bundler.io/) and [Cocoapods](https://cocoapods.org/) installed. Then checkout the repo and run:
+First, make sure you have [`bundler`](http://bundler.io/), [Cocoapods](https://cocoapods.org/) and [nvm](https://github.com/creationix/nvm/#installation) installed. Then checkout the repo and run:
 
 ```
 cd studio
 bundle && bundle exec pod install
 carthage update --new-resolver --platform macOS
+cd LonaStudio/Scripting/Modules/component-to-sketch
+nvm use 8.4
```

Comment:
this can be reduced to just `nvm install 8.4`, and if you commit an `.nvmrc` file to the repo containing “8.4”, just `nvm install` will work.